copy/copy.go

```diff
@@ -19,6 +20,7 @@ import (
 	"github.com/containers/image/transports"
 	"github.com/containers/image/types"
 	"github.com/klauspost/pgzip"
+	"github.com/mattn/go-isatty"
```

Comment:
(Absolutely non-blocking: It’s not _great_ how this uses `syscall.Syscall6` when there is a ~type-safe `golang.org/x/sys/unix.IoctlGetTermios` (used by the progress bar code) — but, oh well, it’s still much better to depend on a reusable package than to open-code it.

[OTOH the progress bar’s `isatty` implementation is not really answering that question at all…])

Reply:
I can check if https://godoc.org/golang.org/x/crypto/ssh/terminal#IsTerminal works when casting the fd to int.

Reply:
Absolutely non-blocking: I have actually looked into this a bit, there’s also `logrus.IsTerminal`; it also uses the `Syscall6` implementation, but we are already depending on that exact package so using it would not bring any net new code.

Not changing this is _perfectly fine_.